tests/test_utils/test_schemaview.py

```diff
@@ -1036,40 +1036,67 @@ def test_enums_and_enum_relationships(schema_view_no_imports: SchemaView) -> Non
     with pytest.raises(ValueError, match='No such enum: "not_an_enum"'):
         view.permissible_value_parent("not_a_pv", "not_an_enum")
 
-    for en, e in view.all_enums().items():
-        if e.name == "Animals":
-            for pv in e.permissible_values:
-                if pv == "CAT":
-                    assert view.permissible_value_parent(pv, e.name) is None
-                    assert view.permissible_value_ancestors(pv, e.name) == ["CAT"]
-                    assert "LION" in view.permissible_value_descendants(pv, e.name)
-                    assert "ANGRY_LION" in view.permissible_value_descendants(pv, e.name)
-                    assert "TABBY" in view.permissible_value_descendants(pv, e.name)
-                    assert "TABBY" in view.permissible_value_children(pv, e.name)
-                    assert "LION" in view.permissible_value_children(pv, e.name)
-                    assert "EAGLE" not in view.permissible_value_descendants(pv, e.name)
-
-                if pv == "LION":
-                    assert "ANGRY_LION" in view.permissible_value_children(pv, e.name)
-
-                if pv == "ANGRY_LION":
-                    assert view.permissible_value_parent(pv, e.name) == ["LION"]
-                    assert view.permissible_value_ancestors(pv, e.name) == ["ANGRY_LION", "LION", "CAT"]
-                    assert view.permissible_value_descendants(pv, e.name) == ["ANGRY_LION"]
+    animals = "Animals"
```

Comment:
made these tests more comprehensive and got rid of the unnecessary iteration / nesting